docs/content/troubleshooting/csi_ug_troubleshooting_node_crash.md

```diff
@@ -9,15 +9,15 @@ When a worker node shuts down or crashes, all pods in a StatefulSet that reside
 
```

Comment:
I think this page title should be changed:
`# Recovering a pod volume attachment from a crashed Kubernetes node` ->
`# Recovering a statefulset pod from a crashed Kubernetes node`

Reply:
Please open a ticket for this to be potentially updated for the next version release. This requires a discussion and has been written as such since first release of this product.

Reply:
I agree this gap is old, and indeed it can wait for next release, np.
but I don't think it requires an offline discussion. I think we can decide about this here, when the time comes

Reply:
per our conversation, i'll change this to "Recovering from a crashed Kubernetes node" in 1.9.0

Reply:
thanks, fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466